recipes/aioboto3/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "aioboto3" %}
+{% set version = "9.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: abac5dcfa871627b7040e6586a69e3359e2dfc4e15dc66135969f2d26fbdcb3b
+
+build:
+  number: 0
+  noarch: python
+  script: rm -f poetry.lock && {{ PYTHON }} -m pip install --no-deps .
+
+requirements:
+  build:
+    - python >=3.7,<3.9
+    - pip
+    - poetry >=1.1.11,<2.0.0a
+    - poetry-dynamic-versioning >=0.13.1,<0.14.0a
+  run:
+    - python >=3.7
+    - aiobotocore >=2.3.0
+    - cryptography >=2.3.1
+    - chalice >=1.24.0
+    - boto3 ==1.17.106
+    - s3transfer >=0.4.0,<0.5.0a
```

```suggestion
    - s3transfer >=0.4.0,<0.7.0a
```
Just to be safe `v0.6.0` is the latest https://github.com/boto/s3transfer/releases/tag/0.6.0